lib/MentionsForm/index.js

```diff
@@ -87,8 +89,12 @@ class MentionsForm extends Component {
   };
 
   updateInputValue = e => {
-    this.setState({ inputValue: e.target.value });
-    this.props.onInputChange(e.target.value);
+    const { value } = e.target;
+    const mentionCount = (value.match(pattern) || []).concat(
```

Comment:
I imagine there is probably a way to combine the two regexes here with an OR operator, so we only need to do the `value.match` once?